.circleci/fetch_merge_commit.sh

```diff
@@ -25,6 +25,11 @@ GREEN() { echo -e "\n\033[0;32m$1\033[0m"; }
 YELLOW() { echo -e "\n\033[0;33m$1\033[0m"; }
 RED() { echo -e "\n\033[0;31m$1\033[0m"; }
 
+if [[ -z "$1" ]]; then
+  echo "Usage: fetch_merge_commit.sh [fetch | merge]"
+  exit 1
+fi
```

Comment:
Now that this script no longer explicitly uses the `merge` command line arg, WDYT about splitting this into two scripts called `compute_merge_commit.sh` (`curl` this in the `Compute Merge Commit` step) and `fetch_merge_commit.sh` (this script, which essentially remains unchanged)?

(The first couple of early-exit sections can be copy-pasted to the extent necessary)

Reply:
Because of the duplication, I actually think they should remain the same… I'mma leave this one to you.

Reply:
Fair deal. You've already gone above and beyond with this PR. 😃

Reply:
Cleanup coming up in #33215.